idea_town/frontend/static-src/styles/_theme-helpers.scss

```diff
@@ -4,10 +4,15 @@ $default: #0c99d5;
 $primary: #6fbe4a;
 $wash: #e2eaed;
 $dark: #56565a;
+$light: #f3f3f3;
+$transparent-white: rgba(255, 255, 255, .6);
+$transparent-black: rgba(0, 0, 0, .3);
+$trasparent-wash: rgba(226, 234, 237, .96);
+$transparent-stick: rgba(255, 255, 255, .96);
```

Comment:
Turns out all the other transparent-* colors were rgb versions of hex colors, so that code got much more readable :-)